build.gradle

```diff
@@ -24,12 +24,19 @@ repositories {
     maven {
         url = 'https://repo.extendedclip.com/content/repositories/placeholderapi/'
     }
+    maven {
+        url = "https://nexus.leonardbausenwein.de/repository/maven-public/"
+    }
 }
 
 dependencies {
     compileOnly "org.spigotmc:spigot-api:1.20-R0.1-SNAPSHOT"
     implementation "com.jeff_media:SpigotUpdateChecker:3.0.3"
     compileOnly 'me.clip:placeholderapi:2.11.5'
+    implementation "de.cubbossa:Translations:3.0.0"
+    implementation "net.kyori:adventure-api:4.11.0"
+    implementation "net.kyori:adventure-platform-bukkit:4.1.2"
+    implementation "net.kyori:adventure-text-minimessage:4.11.0"
```

Comment:
I think Spigot doesn't have MiniMessage by default, you should add it as libraries in plugin.yml